src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  padding: 10px;
+  flex-wrap: wrap;
+  width: $calendar--day-size * 7 + 10px * 2 + $calendar--gap * 7;
+  margin: auto;
+  padding-top: $calendar--day-size * 0.85;
+
+  &__day {
+    display: flex;
+    box-sizing: border-box;
+    height: $calendar--day-size;
+    margin-right: $calendar--gap;
+    margin-bottom: $calendar--gap;
```

Comment:
you can use grid-gap for parent class

Reply:
I tried to do it through gap, but it doesn't work for me =(

Reply:
What exactly didn't work for you?